src/provisioning/Provisioner.ts

```diff
@@ -843,8 +843,15 @@ export class Provisioner {
         }
     }
 
-    // Unlink an IRC channel from a matrix room ID
-    public async unlink(req: ProvisionRequest) {
+    /**
+     * Unlink an IRC channel from a matrix room ID
+     * @param req An Express Request which triggered the action. Its body should contain
```

Comment:
Ahhh, now technically it isn't an express object

Reply:
```suggestion
     * @param req An ExpressJS-Request-like object which triggered the action. Its body should contain
```